src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -534,6 +534,12 @@
   "DownloadClientStatusSingleClientHealthCheckMessage": "Download clients unavailable due to failures: {downloadClientNames}",
   "DownloadClientTransmissionSettingsDirectoryHelpText": "Optional location to put downloads in, leave blank to use the default Transmission location",
   "DownloadClientTransmissionSettingsUrlBaseHelpText": "Adds a prefix to the {clientName} rpc url, eg {url}, defaults to '{defaultUrl}'",
+  "DownloadClientTriblerSettingsAnonymityLevel": "Anonymity level",
+  "DownloadClientTriblerSettingsAnonymityLevelHelpText": "Number of proxies to use when downloading content. To disable set to 0. Proxies reduce download/upload speed. See https://www.tribler.org/anonymity.html",
```

Comment:
please use `{url}` for the help url and pass it in with a `[FieldToken]` on the property. We don't want translators to accidently translate the url.

Reply:
Thank you, i have pushed a fix for the translations.